app/model/editions/EditionsCollection.scala

```diff
@@ -0,0 +1,59 @@
+package model.editions
+
+import java.time.ZonedDateTime
+import play.api.libs.json.Json
+
+import scalikejdbc.WrappedResultSet
+
+case class EditionsCollection(
+    id: String,
+    name: String,
+    prefill: String,
+    isHidden: Boolean,
+    updatedOn: Option[ZonedDateTime],
```

Comment:
Additionally, previously this was `lastUpdated` ... sorry again 🙄

Reply:
There may be genuine reasons for this, just thought I'd spell out the differences